awx/network_ui/views.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2017 Red Hat, Inc
```

Comment:
This file is strange in that it doesn't use any DRF constructs but maybe that's okay since we've simplified?

I'd merge what you have in serializers down into this... This then becomes a temporary set of views until we make this a much larger feature that requires DRF.

Reply:
I've merged serializers into views.